File: programovani/ulohy/files/modelovani-obrazcu-i/obdelnik.cpp
#include <iostream>
using namespace std;


int main() {

    // Deklarace a inicializace
    const int a = 6; // Inicializace strany obdelnika
    const int b = 4; // Inicializace strany obdelnika

    // Cyklus pro namalovani obrazce
    for(int i = 1; i <= a * b; i++) {

        // Vypiseme hvezdicky
        cout << "* ";

        // Pokud jsme dosli na konec radky 
        if(i % a == 0)

            // Vypiseme novou radku
            cout << endl;

    }

}

File: programovani/ulohy/files/modelovani-obrazcu-i/obdelnik.js
// Deklarace a inicializace
const a = 6; // Inicializace strany obdelnika
const b = 4; // Inicializace strany obdelnika
var rad = ""; // Inicalizujeme promennou pro vypsani radky

// Cyklus pro namalovani obrazce
for(var i = 1; i <= a * b; i++) {

    // Pridame do textoveho retezce hvezdicku
    rad += "* ";

    // Pokud jsme dosli na konec radky 
    if(i % a == 0) {

        // Vypiseme radku
        console.log(rad);

        // Smazeme textovy retezec
        rad = "";

    }

}
